vitals/BatteryLimit.java

```diff
@@ -0,0 +1,74 @@
+package vitals;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class BatteryLimit {
+
+	public static String measureUnitDefault = "Celsius";
+	static List<String> unitsTemperature = Arrays.asList("Celsius", "Fahrenheit");

```

Comment:
duplicate "Celsius" declaration.....let first create list and set measurUnitDefault from the list

Reply:
Changed logic. Kindly check